kb/2023-08-11/using_netapp_third_party_storage.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Using NetApp Storage on Harvester
+description: Installation procedure for NetApp Astra Trident CSI Driver
+slug: install_netapp_trident_csi
+authors:
+  - name: Jeff Radick
+    title: Staff Software Engineer
+tags: [harvester]
+hide_table_of_contents: false
+---
+
+Instructions for installing the Netapp Astra Trident CSI driver into a Harvester cluster.
+
+This enables use of NetApp storage systems to store storage volumes usable by virtual machines running in Harvester.
+
+The NetApp storage will be an option in addition to the normal Longhorn storage; it will not replace Longhorn. Virtual machine images will still be stored using Longhorn.
+
+This has been tested with Harvester 1.2.0 and Trident v23.07.0.
+
+This procedure only works to access storage via iSCSI, not NFS.
+
+:::note
+3rd party storage classes (including those based on Trident) can only be used for non-boot volumes of Harvester VMs.
+:::
+
+# Detailed Instructions
+
+We are assuming that before beginning this procedure,
+a Harvester cluster and a NetApp ONTAP storage system
+are both installed and configured for use.
+
+Most of these steps can be performed on any system that has the `helm` and `kubectl` commands installed, and which has network connectivity to the management port of the Harvester cluster.  Let's call this your workstation.  Certain steps must be performed on one or more cluster nodes themselves.  The steps described below should be done on your workstation unless otherwise indicated.
+
+The last step (setting up multipathd) should be done on all nodes,
+after the Trident CSI has been installed.
+
+Certain parameters of your installation will require modification of details in the examples in the procedure given below. This which you may wish to modify include:
```

Comment:
perhaps "Those which you may wish to modify"?

Reply:
right

Reply:
done